Gui/opensim/view/src/org/opensim/view/experimentaldata/ExperimentalMarker.java

```diff
@@ -104,9 +107,11 @@ public JSONObject createDecorationJson(ArrayList<UUID> comp_uuids, MotionDisplay
         StateVector dataAtStartTime = motionDisplayer.getSimmMotionData().getStateVector(0);
         ArrayDouble interpolatedStates = dataAtStartTime.getData();
         int idx = getStartIndexInFileNotIncludingTime();
+        AnnotatedMotion mot = (AnnotatedMotion)motionDisplayer.getSimmMotionData();
+        conversion = mot.getUnitConversion();
```

Comment:
Yes, the reading sets the conversion factor. I just restored the code that was used in 3.3 for conversion. Experimental data is read into AnnotatedMotion (subclas of Storage that annotates the data with meaning) while regular data is read into Storage objects

Reply:
@aseth1 the code was rewritten because it was vtk objects-centric so that was needed for the new visualizer. It would be good to have a variety of trc files so these things don't fall through the cracks